Assets/MixedRealityToolkit.Extensions/SpectatorView/Scripts/UI/RecordingServiceVisual.cs

```diff
@@ -93,17 +93,18 @@ protected void Awake()
                 Debug.LogError("Error: Recording service not set for RecordingServiceVisual");
                 return;
             }
-
-            _recordButton.onClick.AddListener(OnRecordClick);
-            _previewButton.onClick.AddListener(OnPreviewClick);
         }
 
         protected void Update()
         {
             if (_recordingService != null &&
-                _previewButton != null)
+                _recordButton != null &&
+                _recordButton.gameObject != null &&
```

Comment:
Why the GameObject checks?

Reply:
I guess we don't have to. This is my naiveness to Unity showing. I wasn't sure if button components could have null game objects